.gitignore

```diff
@@ -46,5 +46,5 @@ ENIGMAsystem/SHELL/.objs
 MacOS/build/Release/EnigmaXcode.app/Contents/Info.plist
 MacOS/build/Release/EnigmaXcode.app/Contents/Resources/*
 *.stackdump
-emake
+/emake
```

Comment:
why?

Reply:
You were ignoring everything called "emake" (including the emake source folder). Now it just ignores the binary at the repo's root.